QUARKUS-6258.md

```diff
@@ -15,9 +15,16 @@ The list of prioritized features (taken from the issue):
 - Advanced RAG
 - Guardrails 	
 - WebSocket chat integration 	
-- CDI Chat Model.
+- CDI Chat Model
 
-We also need to verify correct usage of OpenAI and Watson.x (optional for the first release) APIs as well as observability (metrics, logging, and telemetry).
+We also need to verify correct usage of OpenAI and Watson.x (optional for the first release) APIs as well as observability (metrics, logging, and telemetry), the latter being a requirement for full support.
+
+### Optional test cases
+These are required only for test-preview status in case Quarkus will go the way of maximum support in te first release:
+- Scoring Model
+- Parsing of documents (PDF, Word, etc)
+- MCP client and server (STDIO and Streamable HTTP )
```

Comment:
wow, can you elaborate what you mean by `MCP client and server (STDIO and Streamable HTTP )`? Because on testing Quarkus MCP server, you would have great deal of work, does it mean that you just want to test some scenarios that use MCP client/server?